arbnode/batch_poster.go

```diff
@@ -1516,7 +1524,11 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		// don't post anything for now
 		return false, nil
 	}
-
+	// If we are checking the validation, set isWaitingForEspressoValidation in the batch segments and re-poll the function until we are ready to post.
+  hasBatchBeenValidated:= b.checkEspressoValidation()
+	if !hasBatchBeenValidated {
```

Comment:
can we add a log here for debugging purposes of devnet and testnet? we can do log cleanup later but it will help until we have a working implementation